helm/charts/hpe-csi-driver/README.md

```diff
@@ -65,57 +50,49 @@ helm repo update
 
 Install the latest chart:
 ```
-# Helm 3
 helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml
-
-# Helm 2, for Nimble, Install with HPENodeInfos CRD's enabled explicitly
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml --set crd.nodeInfo.create=true
 ```
 
+**Note**: values.yaml is optional if no parameters are overridden from defaults.
+
 ### Upgrading the Chart
 
 To upgrade the chart, specify the version you want to upgrade to as below. Please do NOT re-use a full blown `values.yaml` from prior versions to upgrade to later versions. Always use `values.yaml` from corresponding release from [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver)
 
+List the avaiable version of the plugin:
```

Comment:
Sorry for being picky here, is this chart version or app version you list here? We normally don't say "plugin", we refer to the CSI driver as "driver".

Reply:
Also, "available" type.

Reply:
thanks, will modify..:).. i will let 3PAR team review this before merge.